hw1/task1/matrices.py

```diff
@@ -0,0 +1,37 @@
+from vectors import *
+import copy
+
+
+class Matrix:
+    content: List[Vector]
+
+    def __init__(self, first: Vector, *args: Vector):
+        if not all(len(first) == len(vector) for vector in args):
```

Comment:
You should additionally check before this that the number of rows and the number of objects in the first row is not 0, because currently it is possible to play with empty matrices.